core/src/com/unciv/models/ruleset/tile/TileImprovement.kt

```diff
@@ -28,11 +28,17 @@ class TileImprovement : NamedStats() {
 
     fun getTurnsToBuild(civInfo: CivilizationInfo): Int {
         var realTurnsToBuild = turnsToBuild.toFloat() * civInfo.gameInfo.gameParameters.gameSpeed.modifier
-        // todo UNIFY THESE
-        if (civInfo.hasUnique("Worker construction increased 25%"))
-            realTurnsToBuild *= 0.75f
-        if (civInfo.hasUnique("Tile improvement speed +25%"))
-            realTurnsToBuild *= 0.75f
+        for (unique in civInfo.getMatchingUniques("-[]% tile improvement construction time")) {
```

Comment:
Sorry again for being late - thus just food for thought: I wouldn't have hardcoded this to be a bonus - allowing weird mods to slow down workers wouldn't have cost us anything here, just moving the sign to json...